sufia-models/app/actors/curation_concern/generic_work_actor.rb

```diff
@@ -46,7 +46,11 @@ def add_to_collections(new_collection_ids)
       end
 
       #add to new
-      curation_concern.collection_ids = new_collection_ids
+      new_collection_ids.each do |coll_id|
+        collection = Collection.find(coll_id)
+        collection.members << curation_concern
+        collection.save
```

Comment:
@jcoyne Do we really need to save after each assign?  Couldn't this be moved outside the loop?